packages/client/bsconfig.json

```diff
@@ -7,10 +7,6 @@
     {
       "dir": "src",
       "subdirs": true
-    },
-    {
-      "dir": "__tests__",
-      "type": "dev"
```

Comment:
Can remove this now since unit-tests are merged and working again!

EDIT: Or rather, undo these changes